src/pages/RequestCallPage.js

```diff
@@ -279,5 +347,8 @@ export default compose(
             key: ONYXKEYS.REQUEST_CALL_FORM,
             initWithStoredValues: false,
         },
+        inboxCallUserWaitTime: {
+            key: ONYXKEYS.INBOX_CALL_USER_WAIT_TIME,
+        },
```

Comment:
Should add this to the `propTypes` and add a defaultProp for it.

Reply:
Good idea.